jabber.go

```diff
@@ -0,0 +1,112 @@
+package glutton
+
+import (
+	"fmt"
+	"net"
+	"context"
+	"encoding/xml"
+	"bufio"
+	"reflect"
+	"unsafe"
+)
+
+type ServersJabber struct {
```

Comment:
@glaslos Server xml's format is different due to client side portion at jabber service.

server code:
`<?xml version="1.0" encoding="UTF-8"?>
  <servers version="1">
      <server>
          <serverName>Shanghai_VPN</serverName>
          <serverIP>127.0.0.1</serverIP>
      </server>
      <server>
          <serverName>Beijing_VPN</serverName>
          <serverIP>127.0.0.2</serverIP>
      </server>
  </servers>`
client code :
`<?xml version='1.0' ?>
<stream:stream to='192.168.58.137' xmlns='jabber:client'  xmlns:stream='http://etherx.jabber.org/streams' version='1.0'></stream:stream>`